hilgendorf_moveit_config/config/kinematics.yaml

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Did you select something when you configured the chains? iirc you can actually select an empty row in the 'choose your kinematic solver' dropdown. Perhaps that is what happened here?

In any case: yes, you can use:
```
manipulator:
   kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
   kinematics_solver_search_resolution: 0.005
   kinematics_solver_timeout: 0.005
   kinematics_solver_attempts: 3
```
But with a changed name obviously.